packages/charting/src/components/VerticalBarChart/examples/VerticalBarChart.Basic.Example.tsx

```diff
@@ -1,6 +1,8 @@
 import * as React from 'react';
 import { VerticalBarChart, IVerticalBarChartProps } from '@uifabric/charting';
+import { customizable } from 'office-ui-fabric-react/lib/Utilities';
 
+@customizable('VerticalBarChartBasicExample', ['theme', 'styles'])
```

Comment:
I'm curious why this was removed from some examples and added to others?

Reply:
It wasn't needed in some of the examples, since they weren't referencing theme palettes. In others, it was being used but this line was missing, which was breaking the example pages